hack/gen-python-sdk/post_gen.py

```diff
@@ -50,6 +50,8 @@ def _rewrite_helper(input_file, output_file, rewrite_rules):
         lines.append("from kubernetes.client import V1ListMeta\n")
         lines.append("from kubernetes.client import V1Container\n")
         lines.append("from kubernetes.client import V1HTTPGetAction\n")
+        lines.append("from kubernetes.client import V1ManagedFieldsEntry\n")
+        lines.append("from kubernetes.client import V1OwnerReference\n")
```

Comment:
I need to add these lines, not `hack/gen-python-sdk/swagger_config.json` to control `sdk/python/v1beta1/kubeflow/katib/models/__init__.py`
@andreyvelich